lib/class-wp-rest-menu-items-controller.php

```diff
@@ -813,7 +891,18 @@ public function get_item_schema() {
 			'type'        => 'string',
 			'context'     => array( 'view', 'edit', 'embed' ),
 			'arg_options' => array(
-				'sanitize_callback' => 'sanitize_text_field',
+				'sanitize_callback' => static function ( $value ) {
+					$sanitized = sanitize_text_field( $value );
```

Comment:
I'm not sure this function should be called, it was the previous `sanitize_callback`, but AFAICT the title attr is not sanitized with this in Core.